ocm-container.sh

```diff
@@ -17,7 +17,16 @@ EOF
 }
 
 ARGS=()
-BUILD_TAG="latest"
+if [[ -n "${OCM_CONTAINER_TAG}" ]]; then
+  BUILD_TAG="${OCM_CONTAINER_TAG}"
+else
+  BUILD_TAG="latest"
+fi
+if [[ -n "${OCM_CONTAINER_REPO}" ]]; then
+  BUILD_REPO="${OCM_CONTAINER_REPO}"
+else
+  BUILD_REPO="localhost"
```

Comment:
Should we default this to `quay.io/app-sre` instead of localhost?

Reply:
I am also thinking that's a good idea.